avocado/core/loader.py

```diff
@@ -135,10 +137,10 @@ def _str_loaders():
         supported_types = []
         for plugin in self.registered_plugins:
             supported_types.extend(_good_test_types(plugin))
-        # Load plugin by the priority from settings
-        loaders = args.get('loaders', None)
-        if not loaders:
-            loaders = settings.get_value("plugins", "loaders", list, [])
+
+        # Here is one of the few exceptions that can has a hardcoded default
```

Comment:
I think you meant: "Here is one of the few exceptions that has a hardcoded default" ?